src/Raven.Server/Documents/Handlers/Debugging/QueriesDebugHandler.cs

```diff
@@ -17,16 +18,23 @@ public class QueriesDebugHandler : DatabaseRequestHandler
         [RavenAction("/databases/*/debug/queries/kill", "POST", AuthorizationStatus.ValidUser)]
         public Task KillQuery()
         {
+            ExecutingQueryInfo query;
             var name = GetQueryStringValueAndAssertIfSingleAndNotEmpty("indexName");
             var id = GetLongQueryString("id");
+            if (name == "@graph")
```

Comment:
'@graph' should be moved to constants e.g. Constants.Documents.Indexing.GraphIndexName